js-apps/meep-frontend/src/js/util/scenario-utils.js

```diff
@@ -1006,20 +1040,45 @@ export function createDefaultDomain() {
   return domain;
 }
 
-export function createNL(uniqueId, name, element) {
+export function createPoa(uniqueId, name, element) {
   var nl = {
     id: uniqueId,
     name: name,
-    type: NL_TYPE_STR,
+    type: POA_TYPE_STR,
     terminalLinkLatency: getElemFieldVal(element, FIELD_TERM_LINK_LATENCY),
     terminalLinkLatencyVariation: getElemFieldVal(
       element,
       FIELD_TERM_LINK_LATENCY_VAR
     ),
     terminalLinkThroughput: getElemFieldVal(element, FIELD_TERM_LINK_THROUGPUT),
     terminalLinkPacketLoss: getElemFieldVal(element, FIELD_TERM_LINK_PKT_LOSS),
-    physicalLocations: []
+    physicalLocations: [],
+    cellular4gPoaConfig: {}
   };
+
+  return nl;
+}
+
+export function createPoaCell4g(uniqueId, name, element) {
+  var nl = {
+    id: uniqueId,
+    name: name,
+    type: POA_CELL_4G_TYPE_STR,
+    terminalLinkLatency: getElemFieldVal(element, FIELD_TERM_LINK_LATENCY),
+    terminalLinkLatencyVariation: getElemFieldVal(
+      element,
+      FIELD_TERM_LINK_LATENCY_VAR
+    ),
+    terminalLinkThroughput: getElemFieldVal(element, FIELD_TERM_LINK_THROUGPUT),
+    terminalLinkPacketLoss: getElemFieldVal(element, FIELD_TERM_LINK_PKT_LOSS),
+    physicalLocations: [],
+    cellular4gPoaConfig: {}
```

Comment:
I believe you can assign as follows:
cellular4gPoaConfig: {
   cellId: getElemFieldVal(element, FIELD_CELL_ID)
};

Reply:
reworked